### DISCUSSION TITLE
5.2 Game crash

### DISCUSSION THREAD

**Discussion by Simba75013** [Khopa/dcs_liberation#2191] [Q&A] [answered]
Hi there !
Has anyone noted game crashes (DCS frozen) with 5.2 ?
The game doesn't seem to crash in regular SP missions but I cannot play more than 40 50 minutes in Liberation. I did everything, from repair, to desactivate Skynet, fresh install of DCS, drivers ...

I'm experiencing those crashes while I tend to be at the end of the Scenic route campaign, meaning by that there are waaayy less units on the map than at the beginning (and I could fly up to 90min)

I was wondering if there is a common practice to launch missions, something like quit DCS after every turn, launch DCS after Liberation, details not to change after a campaign has been started, some like that ...

Anyone ?

BTW, hardware tested and high end build ; never near any critical hardware limits, like CPU or GPU temps, RAM load, etc ...

**Comment by Starfire13** [comment]
You should always launch DCS after Liberation, not because it prevents crashes but because doing it the other way around will prevent your state.json being written.

But no, I haven't encountered any crashes lately. DCS just doesn't crash much on my end. Perhaps once a year, if that.

**Comment by Simba75013** [comment]
Good to know.
Been doing that today and no crashes.
Do you have to quit DCS after every turn ?

**Reply by Starfire13** [reply]
No, you don't.

**Comment by toadmcfrog** [comment]
Me and a buddy are having this same issue - wondering if you have ever found a fix?  For us, the DCS client always seems to freeze / lock up while on approach for landing, and it happens whether landing on the carrier or on land.  I have had it happen on multiple maps, while running the mission on a dedicated server and while running it locally, etc.  Nothing is written to the log to indicate what might be causing the client to lock up, but it only seems to happen while running Liberation missions.

**Comment by Simba75013** [comment]
Well, I don't think so.
I made sure I launched Liberation first and then DCS, and I did not had any crash after that.
But of course, the farer you go into the campaign the less are there units for the game to load and manage.
Plus, there have been several DCS updates lately.

Additionally, I take more caution on the different units return time and the ability for the carrier to handle all those planes.
I tend to set it up in the way that only the first wave has to land before I do (or S3s because they automaticaly use the elevators).

I did not start another campaign yet, so I'm not really sure the problem is solved though.

The workaround I found to go on is to let the AI do the air missions and focus on the ground ops with Combined Arms.
If you find a way to quit DCS without the need to restart the computer you can manually submit the stats of the mission until the crash.
It is not ideal but it aleviate the frustration of having completed a long mission just to lose everything at the end.

I hope this helps :-)

**Reply by toadmcfrog** [reply]
Thanks for the reply!  I actually found a thread on the DCS forums of a few people having the same problem:  

https://forum.dcs.world/topic/299954-game-freezingnot-responding/

In all of these cases it does seem to be Liberation specific.  Both of the people there also mention the carrier, and having thought about it I didn't see these lock ups while playing a Liberation campaign that didn't have a carrier.  

Very strange.

**Reply by Starfire13** [reply, marked as answer]
Likely cause is running out of memory because Liberation mission are generally quite a bit more taxing than regular missions (due to large number of units). The folks in that thread are talking about running in VR and swapping external views via F2 (and less crashing if there is no F2 view swapping). VR is already resource hungry. Every time one swaps to another plane, all the terrain and objects in a set radius have to be loaded in. If you have preload radius set too high, that would also make matters worse.
